frame/balances/src/lib.rs

```diff
@@ -885,9 +894,8 @@ mod imbalances {
 // its type declaration).
 // This works as long as `increase_total_issuance_by` doesn't use the Imbalance
 // types (basically for charging fees).
-// This should eventually be refactored so that the three type items that do
-// depend on the Imbalance type (TransferPayment, DustRemoval)
-// are placed in their own SRML module.
+// This should eventually be refactored so that the type item that
+// depends on the Imbalance type (DustRemoval) is placed in its own SRML module.
```

Comment:
Will we address https://github.com/paritytech/substrate/issues/2052 in the near future?

Reply:
it's a non-trivial refactor and what we have works, so probably not.